### ISSUE TITLE
snapshots/20210428

### ISSUE THREAD

**Issue by dvdhrm** [osbuild/rpmrepo#4]
This is a tracking issue for the scheduled round of snapshots on 2021-04-28.

**Comment by dvdhrm** [comment]
As of Apr 28 2021 13:53:28, 67 jobs have been scheduled as `daherrma-snapshot-20210428` on AWS Batch. The target suffix is `-20210428`.

**Comment by dvdhrm** [comment]
Failures so far:

```
Job 44: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/AppStream/aarch64/os/repodata/repomd.xml
Job 45: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/BaseOS/aarch64/os/repodata/repomd.xml
Job 46: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/AppStream/ppc64le/os/repodata/repomd.xml
Job 47: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/BaseOS/ppc64le/os/repodata/repomd.xml
Job 48: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/AppStream/s390x/os/repodata/repomd.xml
Job 49: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/BaseOS/s390x/os/repodata/repomd.xml
Job 50: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/AppStream/x86_64/os/repodata/repomd.xml
Job 51: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/BaseOS/x86_64/os/repodata/repomd.xml
Job 52: Status code: 404 for http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.0/compose/RT/x86_64/os/repodata/repomd.xml
```

**Comment by dvdhrm** [comment]
55 snapshots have been created. All 9 snapshots for EL9 failed and were skipped, 2 singleton repos were skipped, and one google-cloud mirror failed. This sums up to the 67 scheduled jobs. Let me know if you would like any of those to be retried now, rather than delayed until the next round.

```sh
curl https://rpmrepo.osbuild.org/v2/enumerate | jq '[.[] | select(test("20210428"))]'
[
  "cs8-aarch64-appstream-20210428",
  "cs8-aarch64-baseos-20210428",
  "cs8-ppc64le-appstream-20210428",
  "cs8-ppc64le-baseos-20210428",
  "cs8-x86_64-appstream-20210428",
  "cs8-x86_64-baseos-20210428",
  "el8-aarch64-appstream-n8.4-20210428",
  "el8-aarch64-appstream-n8.5-20210428",
  "el8-aarch64-appstream-r8.2-20210428",
  "el8-aarch64-appstream-r8.3-20210428",
  "el8-aarch64-baseos-n8.4-20210428",
  "el8-aarch64-baseos-n8.5-20210428",
  "el8-aarch64-baseos-r8.2-20210428",
  "el8-aarch64-baseos-r8.3-20210428",
  "el8-ppc64le-appstream-n8.4-20210428",
  "el8-ppc64le-appstream-n8.5-20210428",
  "el8-ppc64le-appstream-r8.2-20210428",
  "el8-ppc64le-appstream-r8.3-20210428",
  "el8-ppc64le-baseos-n8.4-20210428",
  "el8-ppc64le-baseos-n8.5-20210428",
  "el8-ppc64le-baseos-r8.2-20210428",
  "el8-ppc64le-baseos-r8.3-20210428",
  "el8-s390x-appstream-n8.4-20210428",
  "el8-s390x-appstream-n8.5-20210428",
  "el8-s390x-appstream-r8.2-20210428",
  "el8-s390x-appstream-r8.3-20210428",
  "el8-s390x-baseos-n8.4-20210428",
  "el8-s390x-baseos-n8.5-20210428",
  "el8-s390x-baseos-r8.2-20210428",
  "el8-s390x-baseos-r8.3-20210428",
  "el8-x86_64-appstream-n8.4-20210428",
  "el8-x86_64-appstream-n8.5-20210428",
  "el8-x86_64-appstream-r8.2-20210428",
  "el8-x86_64-appstream-r8.3-20210428",
  "el8-x86_64-baseos-n8.4-20210428",
  "el8-x86_64-baseos-n8.5-20210428",
  "el8-x86_64-baseos-r8.2-20210428",
  "el8-x86_64-baseos-r8.3-20210428",
  "el8-x86_64-google-compute-engine-20210428",
  "el8-x86_64-rt-n8.4-20210428",
  "el8-x86_64-rt-n8.5-20210428",
  "el8-x86_64-rt-r8.2-20210428",
  "el8-x86_64-rt-r8.3-20210428",
  "f33-aarch64-updates-released-20210428",
  "f33-aarch64-updates-released-modular-20210428",
  "f33-x86_64-updates-released-20210428",
  "f33-x86_64-updates-released-modular-20210428",
  "f34-aarch64-devel-20210428",
  "f34-aarch64-devel-modular-20210428",
  "f34-x86_64-devel-20210428",
  "f34-x86_64-devel-modular-20210428",
  "f35-x86_64-rawhide-20210428",
  "f35-x86_64-rawhide-modular-20210428"
]
```
